README.md

```diff
@@ -30,12 +30,12 @@ For the installation of all components we make use of two Ansible roles: the CVM
 https://github.com/galaxyproject/ansible-cvmfs), and a role for adding the EPEL repository (https://github.com/geerlingguy/ansible-role-repo-epel).
 Both repositories are added as a submodule inside the `roles` directory, so make sure to use the `--recursive` option when cloning this repository:
 ```
-git clone --recursive git@github.com:EESSI/cvmfs-layer.git
+git clone --recursive git@github.com:EESSI/filesystem-layer.git
```

Comment:
@bedroge Maybe we should use `https` here?

Reply:
Yes, that's probably better, I've changed this.